examples/flutter_gallery/lib/demo/pesto_demo.dart

```diff
@@ -588,7 +588,7 @@ const List<Recipe> kPestoRecipes = <Recipe>[
       RecipeIngredient(amount: '1 tsp', description: 'Salt'),
     ],
     steps: <RecipeStep>[
-      RecipeStep(description: 'Put in oven'),
+      RecipeStep(duration: '1 min', description: 'Put in oven'),
```

Comment:
Why do you need a time for this step?

Reply:
It's the only step without a duration, but it's not necessary

Reply:
Ah. I guess take it out then.

Reply:
The duration.